lib/src/main/java/com/diffplug/spotless/npm/TsFmtFormatterStep.java

```diff
@@ -33,17 +33,31 @@
 
 public class TsFmtFormatterStep {
 
+	public static final String DEFAULT_TYPESCRIPT_FORMATTER_VERSION = "7.2.2";
+
+	public static final String DEFAULT_TYPESCRIPT_FORMATTER_TYPESCRIPT_VERSION = "3.3.3";
+
+	public static final String DEFAULT_TYPESCRIPT_FORMATTER_TSLINT_VERSION = "5.12.1";
+
 	public static final String NAME = "tsfmt-format";
 
-	public static FormatterStep create(Provisioner provisioner, File buildDir, @Nullable File npm, File baseDir, @Nullable TypedTsFmtConfigFile configFile, @Nullable Map<String, Object> inlineTsFmtSettings) {
```

Comment:
Since this is `public static`, let's keep it for backward compatibility but add an `@Deprecated` and delegate to the new method.